### PR TITLE
introduce execute_bazel scenario

### PR DESCRIPTION
**Pull request by BenTheElder**
ref: https://github.com/kubernetes/test-infra/issues/7023

TODO (follow up): after validating this on the test jobs, roll out to other jobs including CI bazel test/build, test-infra, etc..

/area jobs
/area bazel

### CODE REVIEW

**Comment by BenTheElder** [comment]
I also fixed up the bazel canaries to drop the cache volume / port junk

File: prow/config.yaml [hunk 1]
```diff
@@ -3267,13 +3250,10 @@ presubmits:
         - --repo=github.com/kubernetes-security/$(REPO_NAME)=$(PULL_REFS)
         - --service-account=/etc/service-account/service-account.json
         - --upload=gs://kubernetes-security-prow/pr-logs
+        - --scenario=kubernetes_bazel_ececute
```
Comment (krzyzacy) [inline prow/config.yaml]: execute?

File: prow/config.yaml [hunk 2]
```diff
@@ -3082,11 +3065,10 @@ presubmits:
         - --repo=github.com/kubernetes-security/$(REPO_NAME)=$(PULL_REFS)
         - --service-account=/etc/service-account/service-account.json
         - --upload=gs://kubernetes-security-prow/pr-logs
+        - --scenario=execute
```
Comment (krzyzacy) [inline prow/config.yaml]: bazel_execute?

File: prow/config.yaml (hunk 1 above)
Comment (BenTheElder) [reply prow/config.yaml]: thanks :^)

Comment (BenTheElder) [reply prow/config.yaml]: done

File: prow/config.yaml (hunk 2 above)
Comment (BenTheElder) [reply prow/config.yaml]: done, thanks

**Pull request merged**